app/assets/stylesheets/pages/about.scss

```diff
@@ -0,0 +1,118 @@
+.about-page  {
```

Comment:
Let's try to enforce BEM standards on upcoming iterations for styles. Here's a video for a good intro tutorial on the "how": https://www.youtube.com/watch?v=WH4U4K5W8N8

If you do feel ambitious, you can try to restructure your classes on this PR. Otherwise, make a trello task labelled with "tech debt" to revisit this work

Reply:
That's fair, we can def make a tech debt card and cleanup later